### PR TITLE
Add Cyrus IMAP package and dependencies

### PR DESCRIPTION
**Pull request by citrus-it**

### CODE REVIEW

File: build/cyrus/files/imapd.conf [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# for Sieve scripts.
+sievedir: /var/$(PREFIX)/sieve
+
+###################################################################
+## Important: KEEP THESE IN SYNC WITH cyrus.conf
+###################################################################
+
+lmtpsocket: /var/$(PREFIX)/socket/lmtp
+idlesocket: /var/$(PREFIX)/socket/idle
+notifysocket: /var$(PREFIX/socket/notify
```
Comment (mvwieringen) [inline build/cyrus/files/imapd.conf:50]: I get the feeling there is a typo here shouldn't this read /var/$(PREFIX/socket/notify analog to the ones above ?

Comment (oetiker) [inline build/cyrus/files/imapd.conf:50]:
```suggestion
notifysocket: /var/$(PREFIX)/socket/notify
```

Comment (citrus-it) [reply build/cyrus/files/imapd.conf]: Indeed, thanks!

**Comment by citrus-it** [comment]
I found a problem with the bundled perl modules, putting on hold for now.

**Comment by citrus-it** [comment]
Problem resolved and testing successful, ready for review again

**Review by oetiker — APPROVED** [review]

**Review by hadfl — APPROVED** [review]

**Pull request merged**